curriculum/challenges/english/25-front-end-development/quiz-asynchronous-javascript/66edd630f7666cfa54b404d0.md

```diff
@@ -17,439 +17,519 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is asynchronous programming?
 
 #### --distractors--
 
-Placeholder distractor 1
+A way to execute code in a sequential order.
 
 ---
 
-Placeholder distractor 2
+A method to execute code concurrently without blocking the main thread.
 
 ---
 
-Placeholder distractor 3
+A technique to handle errors in synchronous code.
+
+---
+
+A way to guarantee task order.
```

Comment:
do not add an extra distractor, replace the one that is equal to answer, you should have 4 choices total, 3 distractors and one correct answer